dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23CDIConfigByACPTests.java

```diff
@@ -47,7 +47,13 @@ public static void setup() throws Exception {
 
         // Create the CDIConfigByACP jar that is used in CDIConfigByACP.war,
         JavaArchive cdiConfigByACPJar = ShrinkWrap.create(JavaArchive.class, "CDIConfigByACP.jar");
-        cdiConfigByACPJar.addPackage("com.ibm.ws.jsf23.fat.cdi.common.beans");
+        if (JakartaEE10Action.isActive()) {
+            // Include @Named beans.
+            cdiConfigByACPJar.addPackage("com.ibm.ws.jsf23.fat.cdi.common.beans.faces40");
+        } else {
+            // Include @ManagedBean beans.
+            cdiConfigByACPJar.addPackage("com.ibm.ws.jsf23.fat.cdi.common.beans");
```

Comment:
I think it would be nice to have a separate package for the 2.3  beans (rather than have beans under the common.beans package and then also including the faces40 package within), but I understand that would be more refactoring. 

I think it's fine as is.

Reply:
I fixed it! Thanks!